package-lock.json

```diff
@@ -2506,7 +2528,8 @@
         "ansi-regex": {
           "version": "2.1.1",
           "bundled": true,
-          "dev": true
+          "dev": true,
+          "optional": true
```

Comment:
the changes of ```optional: true``` have been added because the lock file was initially generated with an npm version <=6.5